src/store/store.ts

```diff
@@ -601,16 +601,21 @@ export const useStore = defineStore('store', {
         )
       } else {
         return (
-          // *** FUTURE: update this when Resulting Company Name is implemented
           this.getAmalgamatingBusinessesValid &&
-          !!this.getCorrectNameOption &&
-          this.getNameTranslationsValid
+          // NB - this is the only valid Correct Name Option for short-form amalgamations:
+          (this.getCorrectNameOption === CorrectNameOptions.CORRECT_AML_ADOPT)
+          // NB - there are no name translations for short-form amalgamations
         )
       }
     },
 
     /** Whether all the amalgamation steps are valid. */
     isAmalgamationValid (): boolean {
+      const isCreateShareStructureValid = (
+        this.isAmalgamationFilingHorizontal ||
+        this.isAmalgamationFilingVertical ||
+        this.isCreateShareStructureValid
+      )
```

Comment:
Basically, don't check the share structure valid flag (because the share structure page was not displayed), but we can assume they are correct because we got them from the holding/primary company.